shared/actions/chat2/index.js

```diff
@@ -623,6 +623,11 @@ const loadMoreMessages = (
 
   if (action.type === Chat2Gen.loadOlderMessagesDueToScroll) {
     paginationKey = meta.paginationKey
+    // no more to load
+    if (!paginationKey) {
```

Comment:
if you scroll all the way up this will be null, and we'd still try and load  so we'd load a page of new content on top of the oldest content